docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: '3'
 services:
   telegram_notify_dev:
     container_name: telegram_notify_dev
-    image: cloudblueconnect/connect-extension-runner:26.33
+    image: cloudblueconnect/connect-extension-runner:26.36
```

Comment:
Why you don't move to new schema using a Dockerfile?? As a sample shall look like bootstraped with our CLI